api/opds2.py

```diff
@@ -58,3 +78,74 @@ def feed_links(self):
         return [
             {"href": self.url, "rel": "self", "type": self.OPDS2_TYPE},
         ]
+
+
+class OPDS2API(BaseCirculationAPI):
+    """
+    Generic OPDS2 Circulation APIs
+    This should be a pure implementation with no references to daatasources
+    """
+
+    @classmethod
+    def get_authentication_token(
```

Comment:
Since this isn't used anywhere else, should this function just get moved into `OPDS2TokenAuthenticationFulfillmentProcessor`?

Reply:
Since the {authentication_token} is a more generic OPDS2 workflow I thought having it accessible elsewhere would be better